tests/fixtures/basic/archive/test/law-html/logs/HEAD

```diff
@@ -0,0 +1,10 @@
+0000000000000000000000000000000000000000 64215d0a7b9398be24035d95d574f329fd3140f5 n-dusan <nikolic.dusan.dey@gmail.com> 1686250251 +0200	commit (initial): Initial commit
```

Comment:
Can such Git activity be Git-ignored?

Reply:
Definitely. Added `logs` and hooks to gitignore